frontend/src/components/RequestDialog.vue

```diff
@@ -183,7 +183,7 @@ const isNewTransferDisabled = computed(() => {
 
 const shownError = () => {
   const error = requestSignerError.value || transactionError.value;
-  if (error) {
+  if (error && requestState.value !== RequestState.RequestFailed) {
```

Comment:
Shouldn't the user be able to start a new transfer in the error case? If I'm not mistaken, the `NewTransfer` button stays disabled now.

And not related to this PR, but I think this function should not modify state. It should actually be a computed property to make use of vue's reactivity system. To modify the state below it would be better to have a separate watcher.

Reply:
> Shouldn't the user be able to start a new transfer in the error case? If I'm not mistaken, the `NewTransfer` button stays disabled now.
You are right.. Didn't notice that while implementing.. TY

> And not related to this PR, but I think this function should not modify state. It should actually be a computed property to make use of vue's reactivity system. To modify the state below it would be better to have a separate watcher.

I am not sure if I got exactly what do u mean but talking about Vue, then you are the expert here 😁!
I think you are talking about getting rid of `showError` at all as a function and converting it into a computed property that will be watched to change `requestState` there. Correct me if I'm wrong.
Anyway, I have just used the current implementation without adding fancy things.. So just added a condition to prevent go to `Init` state if a failure is there because in future we will have information in progress screen like transaction Id and such on that the user will need to be there to trap them for further support!

Reply:
Ok, forget about the button. I misread the `isNewTransferDisabled` condition above. The button shouldn't be disabled. So it's fine.

I would wrap the function like this:
`const shownError = computed(() => ...)`

And remove the check from there and add a watcher:
`ẁatch([shownError], () => <here the check and state change>`

(Computed properties are not allowed to modify state.)